src/Compilers/CSharp/Portable/Symbols/Synthesized/TypeSubstitutedLocalSymbol.cs

```diff
@@ -93,6 +93,10 @@ internal override RefKind RefKind
             get { return _originalVariable.RefKind; }
         }
 
+        internal override uint ValEscapeScope => _originalVariable.ValEscapeScope;
+
+        internal override uint RefEscapeScope => _originalVariable.RefEscapeScope;
```

Comment:
Is this possible to cover this with a test?

Reply:
Actually, will replace that with a throw as well. Compiler shouldn't do that for synthesized symbols at all.

Reply:
This is only used during lowering. All escape errors should be reported before that.